deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
+    "noweb":  false,
+    "site":  "My pump.io site using Docker",
+    "owner":  "My name is Docker",
+    "ownerURL":  "http://dockerio.cloudapp.net",
+    "hostname":  "pumpio.example.com",
+    "port":  443,
+    "urlPort": 443,
```

Comment:
I know it is redundant, but I personally use this option because I deploy pump.io behind the nginx proxy so I would like to make visible all available options in this generic config sample.